libs/util/src/lib/calculate-timeout-timer.ts

```diff
@@ -0,0 +1,7 @@
+const averageWPM = 50;
+
+export const calculateTimeoutTimer = async (text: string) => {
+  const averageWordLength = text.length / 5;
```

Comment:
Can you please document the reason behind 5?
We don't want to have magic numbers